### ISSUE TITLE
Phony targets

### ISSUE THREAD

**Issue by JeffreyBenjaminBrown** [zwegner/make.py#3]
I will happily leave behind almost everything I had to learn about Makefiles. But one feature I might miss is the ability to define phony targets. In a makefile, if you say that x depends on y and z, and mark x as "phony", then in order to "make x" it will just ensure that y and z are up to date.

Using make.py I can already fake this, by creating a recipe that merely touches a dummy file (with all such dummies probably kept in a folder called something like "phony make.py targets"). For me to do that is very little trouble, so please don't consider this a high-stakes issue -- but if you've already implemented phony targets I'd love to know how to use them.

**Comment by zwegner** [comment]
Yeah, that's not implemented yet. I remember Matt Craighead (the original author) being conceptually against phony targets, but I don't think they're necessarily bad. It should be fairly straightforward to implement--just a target with no commands that's always rebuilt. I'll likely add a check that the phony target isn't an actual file either, to make sure there's no confusion...
